recipe/meta.yaml

```diff
@@ -1,35 +1,43 @@
 {% set name = "arviz" %}
-{% set version = "0.11.2" %}
+{% set version = "0.15.1" %}
```

Comment:
v0.16.0 was released 3 days ago. It dropped `py<39` support and changed `numpy` pinning in runtime. Could we build the latest version?